CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Confluent's Golang client for Apache Kafka
 
+## v1.9.3
+
+This is a maintenaince release:
+
+ * Added SetSaslCredentials wrapper for librdkafka rd_kafka_sasl_set_credentials.
```

Comment:
The librdkafka function name is not relevant to users of the Go client - it is merely an implementation detail - instead describe what the new method does/allows

Reply:
Done.